exampleConfig.js

```diff
@@ -12,5 +12,6 @@ exports.config = {
     verifyCert: true, // OPTIONAL (boolean), should we attempt to verify the server certificate before allowing communication, default true
     timeout: 10000, // OPTIONAL (integer), number of milliseconds to wait for establishing a connection to Instrumental before giving up, default 10s
     recordCounterRates: true, // OPTIONAL (boolean) whether or not to send ".rate" metrics with counters, default true
```

Comment:
That's the change I tried to make originally - the default is false, but the example is true.  This is not the place, but that is not a hard change to make.